src/utils/jwt.js

```diff
@@ -0,0 +1,16 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv'
+dotenv.config();
+export const signinToken = (params) =>{
+    const token = jwt.sign(params,process.env.secretKey,{expiresIn:process.env.EXPIRES_IN})
```

Comment:
What happens if `EXPIRES_IN` is not defined?

Reply:
It will provide an Error because you are trying to use undefined valiable

Reply:
You should provide a fix for that.